src/elements/element.arc.js

```diff
@@ -89,6 +89,10 @@ module.exports = Element.extend({
 		var pixelMargin = (vm.borderAlign === 'inner') ? 0.33 : 0;
 		var angleMargin;
 
+		if (vm.circumference > Math.PI * 2) {
```

Comment:
I'm not sure what `circumference` values greater than `Math.PI * 2` are supposed to represent. Would it make sense to instead do something like `circumference = Math.min(opts.circumference, Math.PI * 2)` in the doughnut controller?

Reply:
Maybe, if we don't want to supports that use case.

Reply:
Is there any difference in functionality between this and what I described? It's not clear to me what providing a circumference value greater than `Math.PI * 2` would do

Reply:
In a use case where full circle is 100%, visualize value more than 100%. Or in other words allow overlapping to 2nd round. Any help?

Reply:
I think it's confusing because you can't tell the difference between 132% and 32%. If we wanted to support values greater than 100% then probably the clearest way to do it would be via stacks ([example](https://i.stack.imgur.com/7A9dx.gif)). But I think for now I wouldn't try to support it

Reply:
The usecase for >100% filled circles are apparent, when used with animation (as in the [demo linked above](https://codepen.io/kurkle/pen/jRzRom)). You can show that some measure is overfulfilled that way.

Reply:
Ah, ok. That makes sense to me then. Thanks for clarifying